airflow/contrib/operators/aws_sqs_publish_operator.py

```diff
@@ -19,10 +19,26 @@
 
 import warnings
 
-from airflow.providers.amazon.aws.operators.sqs import SQSPublishOperator  # noqa
+from airflow.providers.amazon.aws.operators.sqs import SqsPublishOperator
 
 warnings.warn(
     "This module is deprecated. Please use `airflow.providers.amazon.aws.operators.sqs`.",
     DeprecationWarning,
     stacklevel=2,
 )
```

Comment:
Do we still need/want this here if we have it in the class init now as well?

Reply:
It's like that for other hooks so I left it as is:
https://github.com/apache/airflow/blob/main/airflow/contrib/hooks/gcp_spanner_hook.py
https://github.com/apache/airflow/blob/main/airflow/contrib/hooks/aws_hook.py